scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -2903,10 +2972,12 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
    */
   def templateStats(): List[Stat] = statSeq(templateStat)
   def templateStat: PartialFunction[Token, Stat] = {
+    case token if token.is[Ellipsis] =>
+      ellipsis(1, unquote[Stat])
```

Comment:
1) `Unquote` is handled within `DefIntro`.
2) To preserve code style in this method.